### PR TITLE
chore(travis): Activate semantic-release on node-12 test success

### PR DESCRIPTION
**Pull request by johnjbarton**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -43,4 +43,11 @@ before_script:
...
 script:
-  - npm run travis
\ No newline at end of file
+  - npm run travis
+
+after_success:
+  # run automated release process with semantic-release
+  - if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_EVENT_TYPE" == "push" && "$TRAVIS_NODE_VERSION" == "12" ]]; then
+      npm i --no-save semantic-release@15 @semantic-release/changelog@3 @semantic-release/git@7;
+      semantic-release;
```
Comment (kyliau) [inline .travis.yml]: Is `semantic-release` installed globally? If not I think we need to call `npm run semantic-release`.

File: release.config.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  debug: true,
```
Comment (kyliau) [inline release.config.js]: what does the debug flag do? Could you please add a comment?

File: .travis.yml (hunk 1 above)
Comment (johnjbarton) [reply .travis.yml]: I just used copy/paste from the working case in karma-jasmine:
https://github.com/karma-runner/karma-jasmine/blob/master/.travis.yml#L21

File: release.config.js (hunk 2 above)
Comment (johnjbarton) [reply release.config.js]: done

**Pull request merged**